sklearn/feature_selection/_univariate_selection.py

```diff
@@ -206,6 +206,10 @@ def chi2(X, y):
     contain only **non-negative features** such as booleans or frequencies
     (e.g., term counts in document classification), relative to the classes.
 
+    For continuous data, it is necessary to discretize the values (binning)
+    into intervals before applying this function. You can use methods such
+    as :class:`~sklearn.preprocessing.KBinsDiscretizer`.
```

Comment:
This is going int the right direction, I would simplify the wording:
```suggestion
    If some of your features are continuous, you need to bin them, for example by using
    :class:`~sklearn.preprocessing.KBinsDiscretizer`.
```

Reply:
Yes I understand, thanks.